Edge views are not working properly

Unable to load Edge views in production mode (release), I was not able to reproduce this locally in development mode.

The following issue is raised in the console:

```
vendor.702e1d8dd81e3baa41d3.js:75 Error: [$injector:unpr] Unknown provider: tProvider <- t
https://errors.angularjs.org/1.8.0/$injector/unpr?p0=tProvider%20%3C-%20t
    at vendor.702e1d8dd81e3baa41d3.js:75
    at vendor.702e1d8dd81e3baa41d3.js:75
    at Object.s [as get] (vendor.702e1d8dd81e3baa41d3.js:75)
    at vendor.702e1d8dd81e3baa41d3.js:75
    at s (vendor.702e1d8dd81e3baa41d3.js:75)
    at l (vendor.702e1d8dd81e3baa41d3.js:75)
    at Object.invoke (vendor.702e1d8dd81e3baa41d3.js:75)
    at vendor.702e1d8dd81e3baa41d3.js:75
    at ae (vendor.702e1d8dd81e3baa41d3.js:75)
    at vendor.702e1d8dd81e3baa41d3.js:75 undefined
```

--- a/app/edge/components/associated-endpoints-datatable/associatedEndpointsDatatableController.js
+++ b/app/edge/components/associated-endpoints-datatable/associatedEndpointsDatatableController.js
@@ -1,6 +1,7 @@
 import angular from 'angular';
 
 export class AssociatedEndpointsDatatableController {
+  /* @ngInject */
   constructor($scope, $controller, DatatableService, PaginationService) {
     this.extendGenericController($controller, $scope);
     this.DatatableService = DatatableService;

--- a/app/edge/components/edge-job-form/edgeJobFormController.js
+++ b/app/edge/components/edge-job-form/edgeJobFormController.js
@@ -33,8 +33,6 @@ export class EdgeJobFormController {
     // see https://regexr.com/573i2
     this.cronRegex = /(@(annually|yearly|monthly|weekly|daily|hourly|reboot))|(@every (\d+(ns|us|µs|ms|s|m|h))+)|((((\d+,)+\d+|(\d+(\/|-)\d+)|\d+|\*) ){4,6}((\d+,)+\d+|(\d+(\/|-)\d+)|\d+|\*))/;
 
-    this.onChangeModel(this.model);
-
     this.action = this.action.bind(this);
     this.editorUpdate = this.editorUpdate.bind(this);
     this.associateEndpoint = this.associateEndpoint.bind(this);
@@ -84,6 +82,10 @@ export class EdgeJobFormController {
 
   dissociateEndpoint(endpoint) {
     this.model.Endpoints = _.filter(this.model.Endpoints, (id) => id !== endpoint.Id);
+  }
+
+  $onInit() {
+    this.onChangeModel(this.model);
   }
 }
 

--- a/app/edge/components/edge-stack-endpoints-datatable/edgeStackEndpointsDatatableController.js
+++ b/app/edge/components/edge-stack-endpoints-datatable/edgeStackEndpointsDatatableController.js
@@ -1,6 +1,7 @@
 import angular from 'angular';
 
 export class EdgeStackEndpointsDatatableController {
+  /* @ngInject */
   constructor($async, $scope, $controller, DatatableService, PaginationService, Notifications) {
     this.extendGenericController($controller, $scope);
     this.DatatableService = DatatableService;

--- a/app/edge/components/edit-edge-stack-form/editEdgeStackFormController.js
+++ b/app/edge/components/edit-edge-stack-form/editEdgeStackFormController.js
@@ -1,4 +1,5 @@
 export class EditEdgeStackFormController {
+  /* @ngInject */
   constructor() {
     this.editorUpdate = this.editorUpdate.bind(this);
   }

--- a/app/edge/components/groups-datatable/groupsDatatableController.js
+++ b/app/edge/components/groups-datatable/groupsDatatableController.js
@@ -1,6 +1,7 @@
 import angular from 'angular';
 
 export class EdgeGroupsDatatableController {
+  /* @ngInject */
   constructor($scope, $controller) {
     const allowSelection = this.allowSelection;
     angular.extend(this, $controller('GenericDatatableController', { $scope: $scope }));

--- a/app/edge/views/edge-jobs/createEdgeJobView/createEdgeJobViewController.js
+++ b/app/edge/views/edge-jobs/createEdgeJobView/createEdgeJobViewController.js
@@ -5,6 +5,15 @@ export class CreateEdgeJobViewController {
       actionInProgress: false,
     };
 
+    this.model = {
+      Name: '',
+      Recurring: false,
+      CronExpression: '',
+      Endpoints: [],
+      FileContent: '',
+      File: null,
+    };
+
     this.$async = $async;
     this.$q = $q;
     this.$state = $state;
@@ -44,15 +53,6 @@ export class CreateEdgeJobViewController {
   }
 
   async $onInit() {
-    this.model = {
-      Name: '',
-      Recurring: false,
-      CronExpression: '',
-      Endpoints: [],
-      FileContent: '',
-      File: null,
-    };
-
     try {
       const [groups, tags] = await Promise.all([this.GroupService.groups(), this.TagService.tags()]);
       this.groups = groups;

--- a/app/edge/views/edge-jobs/createEdgeJobView/createEdgeJobViewController.js
+++ b/app/edge/views/edge-jobs/createEdgeJobView/createEdgeJobViewController.js
@@ -1,4 +1,5 @@
 export class CreateEdgeJobViewController {
+  /* @ngInject */
   constructor($async, $q, $state, EdgeJobService, GroupService, Notifications, TagService) {
     this.state = {
       actionInProgress: false,

--- a/app/edge/views/edge-jobs/edgeJob/edgeJobController.js
+++ b/app/edge/views/edge-jobs/edgeJob/edgeJobController.js
@@ -1,6 +1,7 @@
 import _ from 'lodash-es';
 
 export class EdgeJobController {
+  /* @ngInject */
   constructor($async, $q, $state, EdgeJobService, EndpointService, FileSaver, GroupService, HostBrowserService, Notifications, TagService) {
     this.state = {
       actionInProgress: false,

--- a/app/edge/views/edge-jobs/edgeJobsView/edgeJobsViewController.js
+++ b/app/edge/views/edge-jobs/edgeJobsView/edgeJobsViewController.js
@@ -1,6 +1,7 @@
 import _ from 'lodash-es';
 
 export class EdgeJobsViewController {
+  /* @ngInject */
   constructor($async, $state, EdgeJobService, ModalService, Notifications) {
     this.$async = $async;
     this.$state = $state;

--- a/app/edge/views/edge-stacks/createEdgeStackView/createEdgeStackViewController.js
+++ b/app/edge/views/edge-stacks/createEdgeStackView/createEdgeStackViewController.js
@@ -1,6 +1,7 @@
 import _ from 'lodash-es';
 
 export class CreateEdgeStackViewController {
+  /* @ngInject */
   constructor($state, EdgeStackService, EdgeGroupService, EdgeTemplateService, Notifications, FormHelper, $async) {
     Object.assign(this, { $state, EdgeStackService, EdgeGroupService, EdgeTemplateService, Notifications, FormHelper, $async });
 

--- a/app/edge/views/edge-stacks/edgeStacksView/edgeStacksViewController.js
+++ b/app/edge/views/edge-stacks/edgeStacksView/edgeStacksViewController.js
@@ -1,6 +1,7 @@
 import _ from 'lodash-es';
 
 export class EdgeStacksViewController {
+  /* @ngInject */
   constructor($state, Notifications, EdgeStackService, $scope, $async) {
     this.$state = $state;
     this.Notifications = Notifications;

--- a/app/edge/views/edge-stacks/editEdgeStackView/editEdgeStackViewController.js
+++ b/app/edge/views/edge-stacks/editEdgeStackView/editEdgeStackViewController.js
@@ -1,6 +1,7 @@
 import _ from 'lodash-es';
 
 export class EditEdgeStackViewController {
+  /* @ngInject */
   constructor($async, $state, EdgeGroupService, EdgeStackService, EndpointService, Notifications) {
     this.$async = $async;
     this.$state = $state;

